Source/UI/Validator/PhoneNumberValidator.swift

```diff
@@ -1,12 +1,14 @@
 import Foundation
 
 class PhoneNumberValidator: Validator {
-    func validate(text: String?) -> Bool {
-        isEmpty(text: text)
-    }
+    //  https://api-reference.checkout.com/#operation/requestAToken!path=0/phone&t=request
+    private let maxDigitLength = 25
+    private let minDigitLength = 6
```

Comment:
I don't think we should just drop urls in code. If it really is needed, we may at least drop some context to the url.

Again, my impression was the source of truth for validator was supposed to be inside the Checkout module. Why are we adding these constraints here instead of using a single validator?

Reply:
This was a comment from Deepesh to add it. I agree with Deeepsh, URL is a good reference for it.